### ISSUE TITLE
你好，请问你们有在affectnet data测试过准确率吗？谢谢

### ISSUE THREAD

**Issue by aa12356jm** [Open-Debin/Emotion-FAN#4]

**Comment by Open-Debin** [comment]
Hello, the method is designed for sequence data. However, AffectNet is a kind of image data, so we aren't testing the method on AffectNet. But we have conducted another experiment on AffectNet, we finetuning our resnet18, and we achieved beyond 53% on the validation set of the human-annotated part of the data.

**Comment by aa12356jm** [comment]
thanks

**Comment by Open-Debin** [comment]
@aa12356jm 
Merry Christmas, I recently update the Emotion-FAN, new features include data process, environment install, CK+ code, Baseline code, and more detail instructions. Also, you can find the old version directory of Emotion-FAN in the README.md. I hope my new updates can help you greatly. Please see the Emotion-FAN for more details.
